src/Validators/PhoneValidator.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+namespace Portal\Validators;
+
+use Exception;
+
+class PhoneValidator
+
+{
+    public static function validatePhone($phone)
+    {
+        $trimPhone = (trim($phone));
```

Comment:
As discussed, we might want to accept +44 as input, and replace it with a single 0 as the leading digit. This would require small modifications to the regexes above.

Reply:
Done